docker-compose.yml

```diff
@@ -11,9 +11,11 @@ services:
       - mongodb
     environment:
       EDDI_VERSION: 4.8
+      EDDI_JVM_OPTIONS: "-verbose:class -Xlog:gc*=debug:stdout:time,uptime,level -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/dumps/oom.bin"
       EDDI_ENV: production
       EDDI_MEMORY_MIN: 1G
       EDDI_MEMORY_MAX: 1G
+      EDDI_JAVA_ENV_TUNE_BUFFER: "jdk.nio.MaxCachedBufferSize=1048576"
```

Comment:
Is this actually used anywhere? if so, where`?

Reply:
Yes, start_eddi.sh has a generic mechanism to pass `-D` system properties as keys prefixed with  EDDI_JAVA_ENV. 
`env_vars=($(printenv | grep EDDI_JAVA_ENV_))` - this code reads them.

So it's used to tune native buffer cache.